src/Modules/remove.js

```diff
@@ -0,0 +1,19 @@
+import { save } from "./save";
+export function del(list) {
+  const itemList = document.querySelector('.list-items');
+  itemList.addEventListener('click', function (e) {
+    if (e.target.classList.contains('delete')) {
+      const listItem = e.target.closest('li');
+      if (listItem && itemList.contains(listItem)) {
+        itemList.removeChild(listItem);
+        const id = parseInt(e.target.getAttribute('data-id'));
+        const index = list.findIndex(item => item.index === id);
+        if (index !== -1) {
+          list.splice(index, 1);
+          save(list);
+        }
```

Comment:
- [ ] As the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#project-requirements) suggest, deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1). Kindly ensure to implement this. 
- [ ] In addition, you should ensure that the first task has index(1) instead of (0).
![image](https://github.com/As1imwe-Mark/To_Do_List-App/assets/99394374/f8a1d6f8-64d0-405b-8370-3d9e68042870)